lib/puppet/network/http/api/indirected_routes.rb

```diff
@@ -210,7 +210,7 @@ def first_response_formatter_for(model, request, &block)
     return formatter if formatter
 
     raise Puppet::Network::HTTP::Error::HTTPNotAcceptableError.new(
-            _("No supported formats are acceptable (Accept: %{accepted_formats})") % { accepted_formats: formats },
+            _("No supported formats are acceptable (Accept: %{accepted_formats})") % { accepted_formats: formats.map(&:name).join(',') },
```

Comment:
Might be better to list mime types as that’s what is specified at the HTTP layer.

Reply:
Yeah, the "Accept: " part looks like a header, I'd go with the valid values in a header.

Reply:
Or just leave of the "Accept: " part, if the mime part is difficult to get.

Reply:
ah not hard, just `formats.map(&:mime).join(',')`

Reply:
cool, updated